CHANGES.rst

```diff
@@ -1,8 +1,17 @@
+Release 0.3.0 (unreleased)
+--------------------------
+
+New Features in 0.3.0
+~~~~~~~~~~~~~~~~~~~~~
+
+- ``labgrid-client ssh`` now also uses port from NetworkService resource if
+  available
+
 Release 0.2.0 (released Jan 4, 2019)
 ------------------------------------
 
-New Features
-~~~~~~~~~~~~
+New Features in 0.2.0
+~~~~~~~~~~~~~~~~~~~~~
```

Comment:
This line is already under the `Release 0.2.0` heading, is there a reason to mention the release number again?

Reply:
Yes, this is a workaround for duplicate labels, see [here](https://travis-ci.com/Bastian-Krause/labgrid/jobs/170152600#L1072). I'm open for suggestions though.

Reply:
I see. This is due to the usage of the autosectionlabels extension. There is no way around it, I'm fine with the change.

Reply:
Can we disable ``autosectionlabels`` for the changelog? Otherwise this will keep happening with new releases.

Reply:
Disabling autosectionlabels selectively is not supported by the plugin.

Reply:
OK, we could disable autosectionlabels globally and use explicit labels, but that for another time.